contracts/Tokens/VTokens/VBep20Delegator.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.16;
+pragma solidity 0.8.13;
```

Comment:
How do you plan to replace the current deployments in the main net of these contracts (markets)?

Reply:
As @kkirka we are shifting the vtoken to `0.8.13` solc version, so we can just get rid of `vBep20Delegate.sol` WDYS?